readthedocs/projects/managers.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+from django.db import models
+from .ssh import generate_public_from_private_key
+
+
+class SSHKeyManager(models.Manager):
+
+    def create_from_string(self, private_key, project):
+        """
+        Create a SSHKey instance from a private key.
+
+        :param private_key: private key string (usually called ``id_rsa``)
```

Comment:
Nit: `usually called id_rsa` refers to the filename, not the private key. Seems this can probably be removed.